### PR TITLE
gui: Display rate limit (fixes #5320)

### LINKED ISSUE #5320: gui: Display rate limit
Currently a rate limit (global or per device) is nowhere displayed in the GUI. To find out which device is rate limited you need to edit one device after the other to check the settings. 
Here are some ideas how to display it:

Text note
limit-text
Icon
limit-icon
Text line
limit-line
Text info
limit-muted

If no rate limit is set the icon/texts are hidden. 
My preference is icon version

### PR DESCRIPTION
**Pull request by uok**
Display rate limit for own device and connected devices

limit-show

### CODE REVIEW

**Comment by uok** [comment]
Ok, this works nice!
How do I get a new string into the GUI and transifex for translation?

File: gui/default/index.html [hunk 1]
```diff
@@ -551,8 +551,9 @@ <h4 class="panel-title">
                       <a href="#" class="toggler" ng-click="toggleUnits()">
                         <span ng-if="!metricRates">{{connectionsTotal.inbps | binary}}B/s</span>
                         <span ng-if="metricRates">{{connectionsTotal.inbps*8 | metric}}bps</span>
-                        ({{connectionsTotal.inBytesTotal | binary}}B)</span>
```
Comment (imsodin) [inline gui/default/index.html]: Good find. These superfluous spans are missing in other places, e.g. the closing one for 
```
                                                <span tooltip data-original-title="{{localStateTotal.files | alwaysNumber | localeNumber}} {{'files' | translate}}, {{ localStateTotal.directories | alwaysNumber | localeNumber}} {{'directories' | translate}}, ~{{ localStateTotal.bytes | binary}}B">
```
in line 571 (or thereabouts).

File: gui/default/index.html [hunk 2]
```diff
@@ -551,8 +551,9 @@ <h4 class="panel-title">
...
+                        ({{connectionsTotal.inBytesTotal | binary}}B)
                       </a>
+                      <small ng-if="config.options.maxRecvKbps > 0"><br/><i class="text-muted"><span translate>Limit</span>: {{config.options.maxRecvKbps}} KiB/s</i></small>
```
Comment (imsodin) [inline gui/default/index.html]: I am pretty sure `maxRecvKbps` are actually bits, not bytes. And this should follow the rate limit unit, i.e. change unit on toggle as well.

File: gui/default/index.html (hunk 1 above)
Comment (uok) [reply gui/default/index.html]: fixed!

File: gui/default/index.html (hunk 2 above)
Comment (uok) [reply gui/default/index.html]: I entered "123" in the settings (it says KiB/s there) and "123" is displayed in the GUI line I added.

**Comment by calmh** [comment]
Chill until next Wednesday, when magic scripts run and it happens automagically.

Or, look into `go run build.go translate` which updates json files based on the HTML source etc

**Comment by uok** [comment]
ok, then this PR is good to go from my side.

**Comment by uok** [comment]
good to go
Is commit squashing still needed?

**Comment by calmh** [comment]
Squashing happens at merge 👍

**Pull request merged**